src/web/mjs/connectors/templates/Tappytoon.mjs

```diff
@@ -0,0 +1,72 @@
+import Connector from '../../engine/Connector.mjs';
+import Manga from '../../engine/Manga.mjs';
+
+export default class Tappytoon extends Connector {
+    constructor() {
+        super();
+        super.id = undefined;
+        super.label = undefined;
+        this.tags = [];
+        this.url = undefined;
+        this.baseURL = 'https://www.tappytoon.com';
+        this.apiurl = 'https://api-global.tappytoon.com';
+        this.lang = undefined;
```

Comment:
This is used in some places, but never seems to be initialized correctly

Reply:
It's defined correctly